### PR TITLE
tests: add more charms and bundles to large spread suite

### PR DESCRIPTION
**Pull request by lengau**
This covers a broader set of charms, increasing sensitivity to breaking changes.

Manual run of these changes (still including failure from #1215): https://github.com/canonical/charmcraft/actions/runs/5882073645

Progress for #1140 and #1193

### CODE REVIEW

**Review by syu-w — APPROVED** [review]

**Comment by syu-w** [comment]
That's a lot. Maybe you want to increase the kill timeout time?

File: tests/spread/charms/bundle/task.yaml [hunk 1]
```diff
@@ -1,36 +1,45 @@
...
+manual: true
+systems:
+  - ubuntu-22.04-64
+# Run these earlier since they can take a very long time
+priority: 100
+kill-timeout: 60m
+
+environment:
+  BUNDLE/argo: https://github.com/canonical/argo-operators
+  # Needs a different workflow for now, but a target for later enabling.
```
Comment (mr-cal) [inline tests/spread/charms/bundle/task.yaml:11]: Are there any github issues to link these to? (even if it's a generic issue)

File: tests/spread/charms/operator/task.yaml [hunk 2]
```diff
@@ -5,9 +5,29 @@ systems:
...
+  CHARM/indico: https://github.com/canonical/indico-operator
+  CHARM/kafka: https://github.com/canonical/kafka-operator
+  CHARM/kratos: https://github.com/canonical/kratos-operator
+  CHARM/minio: https://github.com/canonical/minio-operator
+  CHARM/mlflow: https://github.com/canonical/mlflow-operator
+  CHARM/mongodb: https://github.com/canonical/mongodb-operator
+  # Disabled because it uses a tox-based workflow
+  # CHARM/mysql: https://github.com/canonical/mysql-operator
+  # Doesn't use charmcraft
+  # CHARM/nats: https://github.com/canonical/nats-operator
```
Comment (mr-cal) [inline tests/spread/charms/operator/task.yaml:23]: Any reason to keep this?

**Review by mr-cal — APPROVED** [review]

File: tests/spread/charms/operator/task.yaml (hunk 2 above)
Comment (lengau) [reply tests/spread/charms/operator/task.yaml:23]: TODOs for later this pulse (same as above).

**Pull request merged**